upstream/upstream_connector.go

```diff
@@ -1,4 +1,4 @@
-package node_builder
+package follower
```

Comment:
I don't think we need a whole new root directory just for this file, maybe we can put it somewhere under the `module` root directory?

Reply:
Fixed: https://github.com/GetElastech/flow-go/commit/bc9e5f1e6457feee807e02649e8b38abae46b8f6